rfc/ipni-w3c.md

```diff
@@ -212,6 +204,27 @@ Invoke `ipni/offer` with the CID for an [inclusion claim] that associates a CAR
 }
 ```
 
+- User checks CAR status and gets UCAN delegation for `assert/location` where subject `with` field is a provider DID (did:web:web3.storage), issuer is a provider DID, and audience is space DID.
+- User wants to publish to IPNI, so they invoke delegated assert/location capability.
+- Invoke `ipni/offer` with the CID for an [inclusion claim] that associates a CAR CID with a CARx-index CID.
+
+**UCAN invocation** example
+
+```json
+{
+  "iss": "did:key:Alice",
+  "aud": "did:web:web3.storage",
+  "att": [{
+    "can": "ipni/offer",
+    "with": "did:key:space", // users space DID
+    "nb": {
+        "inclusion": CID   // inclusion claim CID
```

Comment:
thinking of this in terms of evolution, it seems to me we can ship this without the CAR indexing on client side for write everywhere by simply not having the inclusion claim, and then all that is generated in CARPark.